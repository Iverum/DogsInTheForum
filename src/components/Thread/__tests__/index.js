import React from 'react'
import ReactDOM from 'react-dom'
import renderer from 'react-test-renderer'
import { Thread } from '../index'

test('thread is rendered correctly', () => {
  const component = renderer.create(
    <Thread
      thread={[
        { username: 'Anonymous', text: 'Lorem Ipsum' },
        { username: 'Nonymous', text: 'Dolor set amet' }
      ]}
    />
  )
  let tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
