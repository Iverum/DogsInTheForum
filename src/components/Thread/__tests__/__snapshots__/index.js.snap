exports[`test thread is rendered correctly 1`] = `
<div
  className="twelve columns">
  <section
    className="row post">
    <div
      className="three columns">
      <img
        src="https://placehold.it/175x175" />
      <h3
        className="username">
        Anonymous
      </h3>
    </div>
    <div
      className="nine columns">
      <div
        className="row">
        <div
          className="text"
          dangerouslySetInnerHTML={
            Object {
              "__html": "<p>Lorem Ipsum</p>
            ",
            }
          } />
      </div>
      <div
        className="row" />
    </div>
  </section>
  <section
    className="row post">
    <div
      className="three columns">
      <img
        src="https://placehold.it/175x175" />
      <h3
        className="username">
        Nonymous
      </h3>
    </div>
    <div
      className="nine columns">
      <div
        className="row">
        <div
          className="text"
          dangerouslySetInnerHTML={
            Object {
              "__html": "<p>Dolor set amet</p>
            ",
            }
          } />
      </div>
      <div
        className="row" />
    </div>
  </section>
  <hr />
  <div
    className="row">
    <textarea
      className="u-full-width"
      onChange={[Function]}
      placeholder="I cast magic missile into the darkness!"
      value="" />
    <input
      className="button-primary"
      onClick={[Function]}
      type="submit"
      value="Submit" />
  </div>
</div>
`;
